config/core/test-configs.yaml

```diff
@@ -1650,6 +1657,20 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: asus-C523NA-A20057-coral
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - cros-ec
+      - igt-gpu-i915
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
```

Comment:
None of the kselftest plans are being run at the moment, missing a build with the combination of x86-chromebook and kselftest fragments.